Library/Homebrew/software_spec.rb

```diff
@@ -122,9 +122,12 @@ def option(name, description = "")
         puts "Symbols are reserved for future use, please pass a string instead"
         name = name.to_s
       end
+      unless String === name
```

Comment:
Yeah, if nobody has objections, I plan on doing a PR soon to change `===` to `is_a?` throughout the core codebase to bring it in line with the style guide and make everything audit&style-clean.